README.md

```diff
@@ -24,6 +24,7 @@ $ bash config.sh
 
 ## Team
 This project is maintained by the following people and a bunch of [awesome contributors](https://github.com/MahdyMirzade/dumBash/graphs/contributors).
+=======
```

Comment:
Including "========" under the description, will break the documents, fix it please.